plane/source/docs/quadplane-vtol-tuning-process.rst

```diff
@@ -228,19 +228,40 @@ used to maintain yaw. This can be caused by:
 
 If too much power is needed to maintain yaw then the aircraft could
 lose yaw control during transitions, or could lose roll and pitch
-stability. For larger QuadPlanes it is common to need to deliberately
+stability. The most common symptom is a high hover throttle point, or even the inability to rise into a hover at high throttle stick position. For larger QuadPlanes it is common to need to deliberately
 tilt the motors by a couple of degrees to increase yaw authority.
 
+.. note:: for an X frame type (:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` = 1), 
+    the motors should be tilted outwards. 
+    For an H frame (:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` = 3) they should be tilted inwards.
+
 You should check the amount of thrust being used to maintain yaw by
 looking at the RATE YOut value in your hover logs. If it is over 10%
 (a value of 0.1) when hovering in no wind then you have a problem with
-yaw asymmetry that should be fixed.
+yaw asymmetry that should be fixed. Just hovering in place may look fine 
+and you may not see a problem until you yaw the plane left and right
+to test this.  If you see RATE.YOut increasing while the plane is yawing 
+then you may also see unbalanced motor outputs in the logs. 
```

Comment:
>10% with no pilot yaw input is what has been a warning flag that yaw imbalance is present

Reply:
Agreed, but this added bit is saying that you should add pilot input and see if YOut increases (at least, that's how I read it), which is not a sign of yaw imbalance; it's expected.

Reply:
Well you do say "YOut increasing while you yaw is normal, **increasing to a really high amount is bad**, but I don't know what value to really recommend as a threshold for a problem." 
In the log I showed, it went up to 3.16 which I'm pretty sure would be far higher than any expected valid value - no?

Reply:
The problem is I've seen examples (videos) from vendors of a plane hovering solidly above ground and them thinking their PIDs are tuned. I'm trying to say you need to actually yaw the vehicle to really see if there is a problem  with yaw asymmetry specifically - then look for "extreme" YOut.

Reply:
> In the log I showed, it went up to 3.16 which I'm pretty sure would be far higher than any expected valid value - no?
Yes, that's huge, particularly because I was pretty sure that value was capped at 1...

I get why you want to say something about this. I don't disagree that it's a potential problem that we should talk about in the wiki, and even on this page somewhere. The issue I have is that the image is labelled as "yaw-asymmetry-log.png" in the section called "Yaw Bias", and it's really not an asymmetry or bias. Yaw asymmetry/bias is when you have a large YOut (i.e., YOut > 0.1) when you are in no-wind and without pilot input.

Probably rename the section to something like "Yaw Authority Problems" and make it clear that yaw bias, poor authority, and adverse twist (having the wrong frame type for a boom configuration) are three different things that can cause yaw authority problems.

Reply:
This is very important and clearly and simply stated. @Hwurzburg  could this be included verbatim? Perhaps somewhere around like 240 where the term is used?

> Yaw asymmetry/bias is when you have a large YOut (i.e., YOut > 0.1) when you are in no-wind and without pilot input.

Reply:
apparently, my 'git-foo' yesterday became 'git-poo'...trying to recover after the accidental deletion required a from-scratch start and some things got lost....will carefully redo today